share/wake/lib/system/path.wake

```diff
@@ -124,7 +124,7 @@ target hashcode (f: String): String =
   if reuse !=* "" then reuse else
     def hashPlan cmd  =
       Plan "" cmd Nil Nil "." "" logNever logError logDebug ReRun True Nil (\_ True) hashUsage id id
-    def job = hashPlan ("<hash>", f, Nil) | runJobWith localRunner
+    def job = hashPlan ("<hash>", f, Nil) | setPlanLabel "hash: {f}" | runJobWith localRunner
```

Comment:
I don't know the full logic; will these start showing up in logs/stdout (maybe with a `-v`, or a `--no-tty`)?  The others are relevant enough to the user that I wouldn't mind having them be shown -- though we want to be careful to not wind up with huge CI logs -- but hashing in particular seems like an internal job that we would kind of prefer to keep hidden.